src/DAL/migration/fullSchema.sql

```diff
@@ -30,31 +30,30 @@ CREATE TABLE records
     update_date timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
     source_start_date timestamp with time zone,
     source_end_date timestamp with time zone,
-    max_resolution_deg numeric NOT NULL,
-    min_horizontal_accuracy_ce_90 real,
+    max_resolution_deg numeric NOT NULL CHECK (max_resolution_deg BETWEEN 0.00000009 AND 0.072),
+    min_horizontal_accuracy_ce_90 real CHECK (min_horizontal_accuracy_ce_90 BETWEEN 0.01 AND 4000),
     sensors text COLLATE pg_catalog."default" NOT NULL,
     srs text COLLATE pg_catalog."default" DEFAULT '4326',
     srs_name text COLLATE pg_catalog."default" DEFAULT 'WGS84GEO',
     region text COLLATE pg_catalog."default",
-    classification text COLLATE pg_catalog."default",
+    classification text COLLATE pg_catalog."default" CHECK (classification ~* '^[3-6]$'),
     links text COLLATE pg_catalog."default" NOT NULL,
     type text COLLATE pg_catalog."default" NOT NULL,
     footprint_geojson text COLLATE pg_catalog."default" NOT NULL,
     keywords text COLLATE pg_catalog."default",
     rms text COLLATE pg_catalog."default",
-    scale integer,
+    scale integer, CHECK (scale BETWEEN 0 AND 100000000),
     layer_polygon_parts text COLLATE pg_catalog."default",
     included_in_bests text COLLATE pg_catalog."default",
     discretes text COLLATE pg_catalog."default",
-    max_resolution_meter numeric NOT NULL,
+    max_resolution_meter numeric NOT NULL CHECK (max_resolution_meter BETWEEN 0.01 AND 8000),
     raw_product_data jsonb,
-    product_bbox text COLLATE pg_catalog."default",
-    CONSTRAINT records_pkey PRIMARY KEY (identifier),
-    CONSTRAINT unique_record_values UNIQUE (product_id, product_version, product_type),
-    CONSTRAINT proper_product_version CHECK (product_version ~* '^\d+\.\d{1,2}$'),
+    product_bbox text COLLATE pg_catalog."default" CHECK (product_bbox ~* '^-?(0|[1-9]\\d*)(\\.\\d*)?,-?(0|[1-9]\\d*)(\\.\\d*)?,-?(0|[1-9]\\d*)(\\.\\d*)?,-?(0|[1-9]\\d*)(\\.\\d*)?$'),
```

Comment:
just validate it before merging,
check not valid values and character situation. for example :1s0,90,-1k0,-90 and etc,